genon/preprocessor/facade/legacy/BOK_첨부용.py

```
@@ -0,0 +1,1866 @@
+    for ext in CONVERTIBLE_EXTENSIONS:
+        pdf_path = pdf_path.replace(ext, '.pdf')
+    return pdf_path
+
+
+def install_packages(packages):
+    for package in packages:
+        try:
+            __import__(package)
+        except ImportError:
+            _log.warning(f"{package} 패키지가 없습니다. 설치를 시도합니다.")
+            subprocess.run([sys.executable, "-m", "pip", "install", package], check=True)
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)